src/functions-and-arrays.js

```diff
@@ -10,8 +10,8 @@ function maxOfTwoNumbers(num1, num2) {
 };
```

Comment:
Another good solution would be using the Math.max(num1,num2), you can learn more about it [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max).